node/src/protocol/signature.rs

```diff
@@ -141,7 +143,8 @@ pub struct FailedGenerator {
     pub msg_hash: [u8; 32],
     pub epsilon: Scalar,
     pub delta: Scalar,
-    pub timestamp: Instant,
+    pub sign_request_timestamp: Instant,
```

Comment:
hmm doesn't look like sign_request_timestamp ever gets used. Maybe we just delete the value entirely

Reply:
sign_request_timestamp was used to pass to the generator created afterwards. The generator_timestamp field was not necessary so deleted.